sktime/forecasting/tests/test_neuralforecast.py

```diff
@@ -14,15 +18,22 @@
 y_train, y_test, X_train, X_test = temporal_train_test_split(y, X, test_size=4)
 
 
-@pytest.mark.parametrize("model_class", [NeuralForecastLSTM, NeuralForecastRNN])
+@pytest.mark.parametrize(
+    "model_class", [NeuralForecastLSTM, NeuralForecastRNN, NeuralForecastAutoLSTM]
+)
 @pytest.mark.skipif(
-    not run_test_for_class([NeuralForecastLSTM, NeuralForecastRNN]),
+    not run_test_for_class(
+        [NeuralForecastLSTM, NeuralForecastRNN, NeuralForecastAutoLSTM]
+    ),
     reason="run test only if softdeps are present and incrementally (if requested)",
 )
 def test_neural_forecast_univariate_y_without_X(model_class) -> None:
     """Test with single endogenous without exogenous."""
+    # get params
+    params = model_class.get_test_params()[0]
```

Comment:
This is probably okay, but as far as I know not all estimators have test examples always. I'm not very fond of relying on that behaviour. Also, if we change the test parameters in future it's not necessary that requirements for the tests will continue to satisfy. I'll prefer an explicit parameter specification for testing. Your edits below probably supports my argument. 

This is subjective, so requesting @fkiraly / @benHeid to share thir opinions.